docs/docs/troubleshooting/connection-issues.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Connection Issues
+sidebar_label: Connection Issues
+description: Troubleshooting wireless connection issues of ZMK devices.
+---
+
+## Split Keyboard Halves Unable to Pair
+
+Split keyboard halves will automatically pair with one another, but there are some cases where this breaks, and the pairing needs to be reset, for example:
+
+- Switching which halves are the central/peripheral.
+- Replacing the controller for one of the halves.
+
+These issues can be resolved by flashing a settings reset firmware to both controllers.
+
+:::warning
+
+This procedure will erase all settings, such as Bluetooth profiles, output selection, RGB underglow color, etc.
+
+:::
+
+First, acquire the reset UF2 image file with one of the following options:
+
+#### Option 1: Build Reset UF2 in 'zmk-config'
+
+Find the `build.yaml` file in your `zmk-config` folder and add an additional settings reset build for the board used by your split keyboard. For example assuming that the config repo is setup for nice!nano v2 with Corne, append the `settings_reset` shield to the `build.yaml` file as follows:
+
+```yml
+include:
+  - board: nice_nano_v2
+    shield: corne_left
+  - board: nice_nano_v2
+    shield: corne_right
+  - board: nice_nano_v2
+    shield: settings_reset
+```
+
+Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
+
+#### Option 2: Download Reset UF2 from ZMK's Workflow
+
+1. [Open the GitHub `Actions` tab and select the `Build` workflow](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain+event%3Apush).
+1. Find one of the 'results' for which the core-coverage job was successfully run, indicated by a green checkmark in the core-coverage bubble like the image example below.
+1. From the next page under "Artifacts", download and unzip the `<board_name>-settings_reset-zmk` zip file for the UF2 image.
+
+| ![Successful core-coverage Job](../../docs/assets/troubleshooting/splitpairing/corecoverage.png) |
+| :----------------------------------------------------------------------------------------------: |
+|    An example of a successful core-coverage job which will produce a settings_reset firmware.    |
+
+### Reset Split Keyboard Procedure
```

Comment:
I think you intentionally changed the header levels here compared to "Option" items, which makes sense. But it's a bit awkward without a level three header before this one. Maybe we add a `### Acquiring a Reset UF2` around line 22?

Reply:
Inconsistent casing is annoying, I agree. Do you want to decide on a casing standard? A quick search pulled up [this tool](https://capitalizemytitle.com/style/ap). I'm not sure an issue needs to be raised for that sort of decision, personally.

I've added the suggested header, but be aware that I do want to reorganize this section some more, in particular using tabs for the reset UF2 acquisition bit. Another thing for the future, though!

Reply:
> Inconsistent casing is annoying, I agree. Do you want to decide on a casing standard? A quick search pulled up [this tool](https://capitalizemytitle.com/style/ap). I'm not sure an issue needs to be raised for that sort of decision, personally.

Does "Title Case" up to a certain level of headers (1-3?) and "Sentence case" after would make sense, perhaps? I don't think it's a big deal what it is exactly, but consistency inside and across pages would be nice to have.

> I've added the suggested header, but be aware that I do want to reorganize this section some more, in particular using tabs for the reset UF2 acquisition bit. Another thing for the future, though!

If it's just a structural change, I think it might make sense to add it here but up to you.

Reply:
I agree with title case up to 3 and sentence case after. While not strictly within scope of this PR, I figured I may as well add it to this since it is a trivial change. Also added a note in dev/docs on it. 

Regarding reset UF2, I think I wanted to make more changes at some point but I can't remember what they were (aside from bolding and underlining that both halves need resetting). I added said change and the tabs, so won't go back to it until I remember what I wanted to do.

Reply:
I forgot to mention - the header cases that I did not change are in the FAQ and pre-commit, since I wasn't sure how to handle those. Title case FAQ just looks odd, and pre-commit is a lowercase thing. Also did not change any of the header level 4+ ones, those don't appear in the sidebar to the right so are more annoying to check (and I would need to look up how to change that).

Reply:
Thanks for doing this, it's quite a lot of work! I agree with the decisions on FAQ and pre-commit.

I'll add a commit to handle the 4+ level headers (I'll also squash and rename commits as appropriate).